marine-examples/records/Config.toml

```diff
@@ -1,7 +1,7 @@
 modules_dir = "artifacts/"
 
 [[module]]
-name = "records_effector"
+name = "records_facade"
```

Comment:
`facade` is the last module in the config. So, here `pure` should be renamed. Or order changed. 
Also Nth module can import functions only from modules `1..N-1`. So, the first module in config cannot import any functions from other modules.